.github/workflows/ci.yml

```diff
@@ -831,23 +831,19 @@ jobs:
             - suite: suite-delta-lake-databricks73
               config: hdp3
             - suite: suite-delta-lake-databricks73
-              ignore exclusion if: >-
-                ${{ secrets.DATABRICKS_TOKEN != '' }}
+              config: default
             - suite: suite-delta-lake-databricks91
               config: hdp3
             - suite: suite-delta-lake-databricks91
-              ignore exclusion if: >-
-                ${{ secrets.DATABRICKS_TOKEN != '' }}
```

Comment:
@findinpath @alexjo2144 @nineinchnick 
or should I just delete DATABRICKS_TOKEN secret for now?

Reply:
I would greatly be in favor of removing `DATABRICKS_TOKEN` secret.

Reply:
DATABRICKS_TOKEN removed.